python/chronos/src/bigdl/chronos/pytorch/context_manager.py

```diff
@@ -57,6 +57,9 @@ def __enter__(self):
         # call `torch.set_num_threads` at most once
         self.forecaster.thread_num = set_pytorch_thread(self.thread_num,
                                                         self.forecaster.thread_num)
+        if hasattr(self.forecaster, 'optimized_model_thread_num'):
```

Comment:
In what case a forecaster does not has this `optimized_model_thread_num`? remind me please :)

Reply:
> In what case a forecaster does not has this `optimized_model_thread_num`? remind me please :)

yeah, autoformer forecaster now only has `thread_num`, and I think we don't need to add `optimized_model_thread_num` to it currently.

Reply:
ok make sense